src/components/virtualRepeater/virtualRepeater.js

```diff
@@ -0,0 +1,452 @@
+/**
+ * @ngdoc module
+ * @name material.components.virtualRepeat
```

Comment:
Why are files/directories called `virtualRepeater` while the module is `virtualRepeat` ?
(Is it a concious decision ?)